lib.js

```diff
@@ -12,6 +12,9 @@ const DEFAULT_MODULE_PREFIX = 'react-native';
 const DEFAULT_PACKAGE_IDENTIFIER = 'com.reactlibrary';
 const DEFAULT_PLATFORMS = ['android', 'ios', 'windows'];
 const DEFAULT_OVERRIDE_PREFIX = false;
+const DEFAULT_AUTHOR_GITHUB = 'github_account_name'
+const DEFAULT_AUTHOR_NAME = 'Your Name'
```

Comment:
This might be out of the scope of this pull request, but do you think it would be possible to get the default values from the global Git config?

Reply:
Yea, probably out of scope for this PR but I don't see a reason why that is not possible. Basically we could first default to the `git config --global user.name` (as well as email) and if those are not set, then only use these secondary default values.

Reply:
I agree we should file an issue for it and follow up. Would be a good starter task too imho.